examples/bevy_ggrs/src/box_game.rs

```diff
@@ -45,44 +40,54 @@ pub struct Player {
 }
 
 // Components that should be saved/loaded need to implement the `Reflect` trait
-#[derive(Default, Reflect, Component)]
+#[derive(Default, Reflect, Component, Clone)]
 pub struct Velocity {
     pub x: f32,
     pub y: f32,
     pub z: f32,
 }
 
 // You can also register resources.
-#[derive(Resource, Default, Reflect, Hash)]
+#[derive(Resource, Default, Reflect, Hash, Copy, Clone)]
 #[reflect(Resource, Hash)]
```

Comment:
I wonder if we should remove the unnecessary `Reflect` derives now? Would perhaps make the example easier to understand?

Reply:
What I've done is directly update this `box_game` example to be a complete copy-paste of the original `bevy_ggrs` example, with the bare minimum changes required to make it work with matchbox. I think that would be the best version of this example, since someone can more directly compare the two examples to see how matchbox changes things.